mlflow/version.py

```diff
@@ -2,7 +2,7 @@
 import re
 
 
-VERSION = "2.1.2.dev0"
+VERSION = "2.2.0"
```

Comment:
This version should be `2.2.1.dev0` ?

Reply:
@harupy I remember the releasing pipeline final task should update the version ?

Reply:
Yes, our internal pipeline automatically updates this line. We should not manually modify it. Let's revert.

Reply:
Currently in the GUI it is displayed like this:
![image](https://user-images.githubusercontent.com/86011874/222452748-dfe450c8-0058-43bb-a74a-d1e8806b1a7c.png)

Reply:
I have removed this line.